charts/application/Chart.yaml

```
@@ -7,4 +7,4 @@ maintainers:
   - name: MediaMarktSaturn
     url: https://github.com/MediaMarktSaturn
 appVersion: 1.0.0
-version: 1.11.1
+version: 1.11.2
```

```suggestion
version: 1.12.0
```

because it's a new feature?